src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,27 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends AbstractManager {
+
+    private double budget;
+
+    public double getBudget() { return budget;}
+
+    public FixedBudgetManager(String name, double salary, int budget) {
+        super(name, salary);
+        this.budget  = budget;
+    }
+
+    @Override
+    public boolean canHireEmployees() {
+        return getSalariesSum() <= budget;
+    }
+
+    private Double getSalariesSum() {
+        return getEmployees().stream().map(AbstractEmployee::getSalary).reduce( 0.0, ((a, b) -> a+b));
```

Comment:
- \+ za użycie streamów 😄 
- Jak używasz streamów to stosuj takie formatowanie można znaleźć pluginy do intellija żeby sam Ci poprawiał na takie formatowanie:
```
private Double getSalariesSum() {
        return getEmployees().stream()
                .map(AbstractEmployee::getSalary)
                .reduce(0.0, ((a, b) -> a + b));
    }
```
dużo ładniej to wygląda:)